README.md

```diff
@@ -33,12 +33,6 @@ $ npm install raw-loader --save-dev
 
 Then add the loader to your `webpack` config. For example:
 
-**file.js**
-
-```js
-import txt from './file.txt';
-```
-
```

Comment:
Why you remove this?

Reply:
Because you have configed `.txt` file in `webpack.config.js` so that when importing this type of file, webpack uses `raw-loader` . At the same time, `raw-loader` can import other types of files as String, as long as you add raw-loader! prefix to tell webpack that you want to use `raw-loader` to import `.css` or `.html` file.

Reply:
@uniqChenjun somebody can waht import all txt files using `raw-loader` so this section for these developers, don't need change this

Reply:
Please check https://github.com/uniqChenjun/raw-loader-demo1
you will see the difference

Reply:
It is expected (https://github.com/webpack/docs/wiki/loaders). See:
```js
import txt2 from 'raw-loader!./test.txt';
```

You run `raw-loader` for `test.txt` because you add `raw-loader` for `txt` in configuration when you run `raw-loader` again when you use `raw-loader!./test.txt`. How it should be fixed:
```
import txt2 from '!!raw-loader!./test.txt'; // !! -disable all loaders from configuration.
```

Reply:
I will update documentation about this. Closse PR because it is invalid usage.

Reply:
Issue about doc https://github.com/webpack/webpack.js.org/issues/2804

Reply:
PR https://github.com/webpack-contrib/raw-loader/pull/65